com.adsbynimbus.nimbus/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "com.adsbynimbus.nimbus",
```

Comment:
@JasonSznol nevermind this must have been an error doing the Timehop name migration.

Reply:
+1